module-apps/application-settings/windows/network/SimCardsWindow.cpp

```diff
@@ -92,8 +150,28 @@ namespace gui
             nullptr,
             gui::option::SettingRightItem::ArrowWhite));
 
-        navBar->setText(nav_bar::Side::Center, utils::translate(style::strings::common::select));
-
         return optList;
     }
+
+    bool SimCardsWindow::createSwitchSimAction(const Store::GSM::SIM selectedSim) const
+    {
+        if (selectedSim == Store::GSM::SIM::SIM1) {
```

Comment:
A ternary operator (`? :`) can be used here.

Reply:
done